The example for `login_required` error isn't compiled

### Describe the problem
The [example](https://github.com/auth0/auth0-angular/blob/d7b52ce31d4756993f37da4c9ce4dbf273e41fde/EXAMPLES.md) for `login_required` error isn't compiled.

I get the following error:
```
Error: src/app/app.component.ts:15:32 - error TS2304: Cannot find name 'GenericError'.

15       filter(e => e instanceof GenericError && e.error === 'login_required'),
```

### What was the expected behavior?

The code is succefully compiled

### Reproduction

Ccompile the following code (html template is any)
``` typescript
import { Component, OnInit } from '@angular/core';
import { AuthService } from '@auth0/auth0-angular';
import { filter, mergeMap } from 'rxjs/operators';

@Component({
  selector: 'app-root',
  templateUrl: './app.component.html'
})
export class AppComponent implements OnInit {

  constructor(private readonly auth: AuthService) {}

  ngOnInit(): void {
    this.auth.error$.pipe(
      filter(e => e instanceof GenericError && e.error === 'login_required'),
      mergeMap(() => this.auth.loginWithRedirect())
    ).subscribe();
  }

}
```

### Environment

> Please provide the following:

- auth0-agular: **2.0.1**
- angular: **15.1**

--- a/projects/auth0-angular/src/public-api.ts
+++ b/projects/auth0-angular/src/public-api.ts
@@ -23,4 +23,11 @@ export {
   InMemoryCache,
   IdToken,
   User,
+  GenericError,
+  TimeoutError,
+  MfaRequiredError,
+  PopupTimeoutError,
+  AuthenticationError,
+  PopupCancelledError,
+  MissingRefreshTokenError
 } from '@auth0/auth0-spa-js';

